akvo/rsr/models/organisation.py

```diff
@@ -321,7 +321,31 @@ def content_owned_organisations(self):
             return queryset
 
     def __unicode__(self):
-        return self.name
+        return self.get_name
+
+    @property
+    def get_name(self):
```

Comment:
This is smart. :+1: 

But, using `get_name` as a property seems a little weird. It would be better if we can come up with a nicer name for the property - though, I can't think of anything, right now.